### DISCUSSION TITLE
DNS provider - exec. Argument "timeout"

### DISCUSSION THREAD

**Discussion by StackAls** [go-acme/lego#1935] [Q&A] [answered]
Hello.
I write to my provider script with arguments: "present", "cleanup" and "timeout".
When I run my script with my parameters - all Ok. DNS record create for "present" and delete for "cleanup" and "timeout" print 
```
{"timeout": 2700,"interval": 5}
{"timeout": 2695,"interval": 5}
{"timeout": 2690,"interval": 5}
...
```
But if I run on traefik - [documentation](https://go-acme.github.io/lego/dns/exec/)
```
EXEC_PATH=./update-dns.sh \
	lego --email you@example.com \
	--dns exec \
	--domains my.example.org run
```
My timeout does not apply - used default timeout

**Comment by ldez** [comment, marked as answer]
Hello,

There is a problem with the documentation: timeout and interval are not related to a command `timeout` but to env vars (https://go-acme.github.io/lego/dns/exec/#additional-configuration)

**Reply by StackAls** [reply]
Thanks a lot - it works!!
